mothertongues/config/models.py

```diff
@@ -307,10 +307,10 @@ class DictionaryEntry(BaseModel):
     example_sentence_definition: Optional[List[str]] = []
     """The example sentence definitions associated with the entry"""
 
-    example_sentence_audio: Optional[List[Audio]] = []
+    example_sentence_audio: Optional[List[Audio|List[Audio]]] = []
```

Comment:
If we allow `example_sentence_audio` to be `List[List[Audio]]` then we need to also update the `ParserTargets` otherwise we won't be able to define `List[List[Audio]]` without passing the values in directly I think, right?

Reply:
It's not obvious at first  but no, because the nested lists are handled by a `listof` target which has type `dict`.

Reply:
Ah, right, that would be true for the JSON parser, but not for the spreadsheet parsers I don't think.

Reply:
Oh, right - in that case we would need the nested list, one could imagine a spreadsheet with columns example1_audio1, example1_audio2, example2_audio_1, etc, etc